### ISSUE TITLE
Sqlite3 Support?

### ISSUE THREAD

**Issue by derekchiang** [eaigner/hood#56]
Hello Erik,

First of all, thank you very much for this awesome package!

I'm developing a web application and I'm strongly considering using hood.  However I will be using Sqlite3, which isn't currently supported.  But I notice that there is a branch "sqlite3", so I'm just wondering if you have been working on sqlite3 support, and if so, would you consider it stable enough to be used in production?  Can I just put splite3.go into the master branch and expect everything to work?

Specifically, I will just need to be able to create/drop tables, create/delete/query items, and ideally declare has-many and one-to-one relationships.  Are these functionalities currently there?

Thank you for your time.

Best,
Derek

**Comment by eaigner** [comment]
No, `sqlite3.go` lags behind ands is not compatible with the master. I currently do not work on sqlite support. It also requires some workarounds to satisfy the dialect interface because sqlite doesn't support all SQL commands, like e.g. altering columns.

**Comment by derekchiang** [comment]
I see.  Thank you!

Date: Tue, 26 Mar 2013 01:20:52 -0700
